axum-core/src/extract/request_parts.rs

```diff
@@ -94,6 +94,16 @@ where
                 .await
                 .map_err(FailedToBufferBody::from_err)?,
             Some(DefaultBodyLimitKind::Limit(limit)) => {
+                let content_length: usize = req
+                    .headers()
+                    .get(http::header::CONTENT_LENGTH)
+                    .and_then(|val| val.to_str().ok()?.parse().ok())
+                    .unwrap();
+
+                if content_length > limit {
+                    todo!()
+                }
```

Comment:
Sorry but I don't think we should add this in this PR. I'd rather have that split up. I think if this PR just removes `ContentLengthLimit` then thats best.